OpenUtau.Core/DawIntegration/DawManager.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using K4os.Hash.xxHash;
+using OpenUtau.Core.Ustx;
+using OpenUtau.Core.Util;
+using Serilog;
+
+namespace OpenUtau.Core.DawIntegration {
+    public class DawManager : SingletonBase<DawManager>, ICmdSubscriber {
+        public DawClient? dawClient = null;
+        CancellationTokenSource? renderCancellation = null;
+        private Debounce sendLayoutDebounce = new Debounce();
+        private Debounce sendAudioDebounce = new Debounce();
+
+        private DawManager() {
+            DocManager.Inst.AddSubscriber(this);
+        }
+
+        public void OnNext(UCommand cmd, bool isUndo) {
+            if (cmd is UNotification && !(
+                cmd is DawConnectedNotification ||
+                cmd is PartRenderedNotification ||
+                cmd is VolumeChangeNotification ||
+                cmd is PanChangeNotification
+                )) {
+                return;
+            }
+
+            sendLayoutDebounce.Do(TimeSpan.FromSeconds(1), async () => {
+                await UpdateUstx();
+                await UpdateTracks();
+            });
+            sendAudioDebounce.Do(TimeSpan.FromSeconds(5), async () => {
```

Comment:
- The interval:

I thought that creating hash for every audio causes high CPU usage, so I debounced to 5 seconds.
Should this be configurable?